app/helpers/puppetclasses_and_environments_helper.rb

```diff
@@ -28,12 +28,21 @@ def pretty_print(classes)
       if (mod = klass.gsub(/::.*/, ""))
         hash[mod] ||= []
         hash[mod] << klass
-      else
-        next
       end
     end
     hash.keys.sort.map do |key|
-      link_to key,{}, {:remote => true, :rel => "popover", :data => {"content" => hash[key].sort.join('<br>').html_safe, "original-title" => key}}
+      @classes_str = "classes"
+      @num = hash[key].size
+      @obj_name = @num > 1 ? @classes_str : @classes_str.singularize
+      link_to key, {}, { :remote => true,
+                         :rel => "popover",
+                         :data => {
+                           "content" => hash[key].sort.join('<br>').html_safe,
+                           "original-title" => _("%{name} has <span class='label label-info'>%{num}</span> %{class_es}") % {:name => key, :num => @num, :class_es => @obj_name},
```

Comment:
Can you move the HTML into the %{num} please so it isn't in the translated string?  You might need to call .html_safe on the string so it doesn't get escaped.

Something like:

    %{name} has %{num} class

and

    :num => "<span class='label label-info'>#{@num}</span>"

(taking into account what I said above too)